dcos/util.py

```diff
@@ -325,16 +325,20 @@ def _format_validation_error(error):
     """
 
     error_message = _hack_error_message_fix(error.message)
+    path = '.'.join([str(path) for path in error.absolute_path])
 
-    match = re.search("(.+) is a required property", error_message)
-    if match:
+    req_match = re.search("(.+) is a required property", error_message)
+    url_match = re.search("^(https{0,1}://).+", str(error.instance))
+
+    if req_match:
         message = 'Error: missing required property {}.'.format(
-            match.group(1))
+            req_match.group(1))
+    elif path == "core.dcos_url" and not url_match:
```

Comment:
I would really prefer we don't merge code with unnecessary one-offs like this, and in general that doesn't meet our design standards.

But @ssk2 is the reviewer, so I'll let him decide.

I also don't understand why this error checking is in a formatting function